app/services/risc_destination_updater.rb

```diff
@@ -24,27 +24,36 @@ def remove
     eventbridge_client.delete_connection(name: connection_name) if connection_exists?
   end
 
+  def issuer_slug
+    service_provider.issuer.gsub(/[^.\-_A-Za-z0-9]/, '_')
```

Comment:
Could use [`ActiveSupport::Inflector.parameterize`](https://api.rubyonrails.org/v6.1.3.2/classes/ActiveSupport/Inflector.html#method-i-parameterize) here?

```suggestion
    ActiveSupport::Inflector.parameterize(service_provider.issuer, separator: '_')
```

Or if we'd be fine with dashes as the separator:

```suggestion
    service_provider.issuer.parameterize
```

Reply:
This regex came from inverting the regex in error message I got from the API through trial and error, so I figure it's easier to maintain as-is?